### PR TITLE
Make seamless intergration block clickable

### PR DESCRIPTION
**Pull request by alygee**
## Description

close [964](https://github.com/epicmaxco/vuestic-ui/issues/964)

 - moved `Seamless` component to another directory
 - added components symbolizing other frameworks

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: packages/docs/src/components/landing/Seamless/Seamless.vue [hunk 1]
```diff
@@ -89,7 +72,19 @@
...
+
+@Options({
+  components: {
+    SeamlessAnotherButton,
+    SeamlessAnotherSelect,
+    SeamlessAnotherRangeSlider,
+    SeamlessAnotherCheckbox
+  }
+})
 export default class Seamless extends Vue {
```
Comment (asvae) [inline packages/docs/src/components/landing/Seamless/Seamless.vue:88]: Can you rename this component to 2 words? SeamlessIntegration maybe?

File: packages/docs/src/components/landing/Seamless/Seamless.vue [hunk 2]
```diff
@@ -5,7 +5,6 @@
         <h2 class="seamless__title">{{$t('landing.seamless.title')}}</h2>
```
Comment (asvae) [inline packages/docs/src/components/landing/Seamless/Seamless.vue:5]: image

Can you remove shadow from this component?

**Review by asvae — APPROVED** [review]
Looks good!

File: packages/docs/src/components/landing/Seamless/Seamless.vue (hunk 1 above)
Comment (alygee) [reply packages/docs/src/components/landing/Seamless/Seamless.vue]: Renamed to SeamlessIntegration

File: packages/docs/src/components/landing/Seamless/Seamless.vue (hunk 2 above)
Comment (alygee) [reply packages/docs/src/components/landing/Seamless/Seamless.vue]: ok

**Pull request merged**